checkstyle-tester/README.md

```diff
@@ -118,14 +118,21 @@ Diff Regression projects: {{URI to projects-to-test-on.properties}}
 
 Diff Regression config: {{URI to my_checks.xml}}
 
-If you want to generate [Difference Report with Different Base and Patch Config]
-(./README_MANUAL_EXECUTION.md#difference-report-with-different-base-and-patch-config)
+If you want to generate [Difference Report with Different Base and Patch Config](./README_MANUAL_EXECUTION.md#difference-report-with-different-base-and-patch-config)
```

Comment:
link was broken because of whitespace between `]` and `(`

Reply:
interesting. mdl should report it.